pkg/integrations/mongodb_exporter/mongodb_exporter.go

```diff
@@ -34,7 +43,7 @@ func (c *Config) Name() string {
 func (c *Config) InstanceKey(_ string) (string, error) {
 	u, err := url.Parse(string(c.URI))
 	if err != nil {
-		return "", fmt.Errorf("could not parse url: %w", err)
+		return "", fmt.Errorf("could not parse mongodb_uri: %s. error: %w", string(c.URI), err)
```

Comment:
This could potentially leak passwords as `c.URI` is a secret

Reply:
Actually the previous code was also leaking the secret sometimes. Can you change it to `fmt.Errorf("could not parse mongodb_uri: %w", errors.Unwrap(err))`?

Reply:
Good catch, will apply the suggestion.